### PR TITLE
tw29615639 Set up route and method for creating a job to send verification email.

### PR DESCRIPTION
**Pull request by earthday47**
https://kanopi.teamwork.com/app/tasks/29615639

https://auth0.com/docs/api/management/v2/jobs/post-verification-email

### CODE REVIEW

File: src/Controller/Auth0Controller.php [hunk 1]
```diff
@@ -49,6 +50,13 @@ final class Auth0Controller extends ControllerBase {
    */
   protected RequestStack $requestStack;
 
+  /**
+   * Drupal\Core\Session\AccountProxy definition.
+   *
+   * @var \Drupal\Core\Session\AccountProxyInterface
+   */
+  protected $currentUser;
```
Comment (sean-e-dietrich) [inline src/Controller/Auth0Controller.php]:
```suggestion
  protected AccountProxyInterface $currentUser;
```

File: src/Service/Auth0Service.php [hunk 2]
```diff
@@ -557,4 +557,23 @@ public function getSalesforceId(string $auth0Id) {
...
+   *
+   * @param string $auth0Id
+   *   Auth0 ID.
+   *
+   * @return bool
+   *   Return true if successful, false if issues.
+   */
+  public function createSendVerificationEmail(string $auth0Id): bool {
+    $response = $this->getManager()->management()->jobs()->createSendVerificationEmail($auth0Id);
+    if ($response->getStatusCode() == 201) {
```
Comment (sean-e-dietrich) [inline src/Service/Auth0Service.php]: Can this be simplified to the following?

```php
return $response->getStatusCode() === 201;
```

File: src/Controller/Auth0Controller.php (hunk 1 above)
Comment (earthday47) [reply src/Controller/Auth0Controller.php]: Done

File: src/Service/Auth0Service.php (hunk 2 above)
Comment (earthday47) [reply src/Service/Auth0Service.php]: Sure!

**Pull request merged**